### ISSUE TITLE
Mroonga and TokuDB

### ISSUE THREAD

**Issue by the-dunnock** [mroonga/mroonga#93]
Known this for long time and was going to debug this will all tracing etc... but my skills ain't that good and clearly learning of it takes me longer than I suspected.

So an issue, when trying to create table with TokuDB using wrapper mode of Mroonga. The connection is simply lost and nothing shows up in error logs.

I do have suspicion why this happens, number 1 guess is that the TokuDB table structure is save to data/ directory and not in to data/db_name/.

If can find out what is causing this issue it would help a lot of people who want to use TokuDB but have to use InnoDB due to not having fulltext index support.

**Comment by kou** [comment]
What Mroonga version are you using?
I can confirm that the following SQL works well on MariaDB 10.0.8 + Mroonga 5.11:

```sql
DROP TABLE IF EXISTS memos;
CREATE TABLE memos (
  id int NOT NULL PRIMARY KEY,
  content TEXT,
  FULLTEXT INDEX (content)
) ENGINE=Mroonga COMMENT='engine "TokuDB"';

INSERT INTO memos VALUES (1, 'Hello');

SELECT * FROM memos;
-- +----+---------+
-- | id | content |
-- +----+---------+
-- |  1 | Hello   |
-- +----+---------+


SELECT * FROM memos
  WHERE MATCH(content) against ('+hello' IN BOOLEAN MODE);
-- +----+---------+
-- | id | content |
-- +----+---------+
-- |  1 | Hello   |
-- +----+---------+
```

**Comment by kou** [comment]
> I can confirm that the following SQL works well on MariaDB 10.0.8 + Mroonga 5.11:

Note that I didn't use Mroonga bundled in MariaDB.

**Comment by the-dunnock** [comment]
Hi, Sorry for late reply
This is the actual error: ERROR 2013 (HY000): Lost connection to MySQL server during query

Just tried with 10.0.22 and apparently this is fixed on 10.1.8. So this was useless report, did not try on 10.1* before. So you can close the issue.

**Comment by kou** [comment]
Thanks for your comment.
I close this issue.
